### ISSUE TITLE
Add tvOS example project

### ISSUE THREAD

**Issue by jessesquires** [Instagram/IGListKit#138]
Depends on #118 and #137.

Let's get a basic tvOS example setup.

Let's put this in a top-level dir, `Examples-tvOS/`

**Comment by jessesquires** [comment]
You'll need to branch off of `tvOS-support` until #137 is merged.

https://github.com/Instagram/IGListKit/tree/tvOS-support

**Comment by Sherlouk** [comment]
Sounds like a plan!

Currently working on creating the base project, will push that and link once that step is done

**Comment by Sherlouk** [comment]
@jessesquires Yes/No to renaming 'Examples/' to 'Examples-iOS/'?

**Comment by Sherlouk** [comment]
For visibility: https://github.com/Sherlouk/IGListKit/tree/tvOS-example

**Comment by jessesquires** [comment]
👍 

Yeah, I was going to suggest that -- `Examples-iOS/` -- let's do that in a separate PR. cc @rnystrom

**Comment by Sherlouk** [comment]
Idea 1 foiled by the lack of #3! Gutted!

Over to plan B haha

**Comment by Sherlouk** [comment]
@jessesquires https://github.com/Sherlouk/IGListKit/tree/tvOS-example

Created a Demo Chooser interface as well as a 'Nested Adapter' example.
Took the concept of the TV show chooser with carousels (would usually have images, but just a label for now).

Mind having a look and giving feedback please? Thanks!

cc: @rnystrom
